doc/devdocs/modules/launcher/plugins/program.md

```diff
@@ -11,7 +11,7 @@ There are broadly two different categories of applications:
 ### [UWP](src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs)
 - The logic for indexing Packaged applications is present within the [`UWP.cs`](src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs) file.
 - There can be multiple applications present within a package. The [`UWPApplication.cs`](src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWPApplication.cs) file encapsulates the properties of a packaged application.
-- To index packaged applications, the `PackageManager` retrives all the packages for the current user and indexes all the applications.
+- To index packaged applications, the `PackageManager` retrivies all the packages for the current user and indexes all the applications.
```

Comment:
retrieves is correct. **retrievies** is not a word.

Reply:
Oh, yes.